pkg/sql/sem/tree/create.go

```diff
@@ -179,8 +179,9 @@ func (node *CreateIndex) Format(ctx *FmtCtx) {
 		ctx.WriteString(")")
 	}
 	if node.Predicate != nil {
-		ctx.WriteString(" WHERE ")
+		ctx.WriteString(" WHERE (")
```

Comment:
I think the parens should be added only if `ctx.HasFlags(FmtPGCatalog)`.

Reply:
done